core/src/main/java/net/adoptopenjdk/icedteaweb/client/console/JavaConsole.java

```diff
@@ -452,47 +440,20 @@ public void run() {
         });
     }
 
-    public void hideConsoleLater() {
```

Comment:
This is really unused?

Reply:
At least the compiler does not complain about a missing method and a search over all code does not find any other hit for the string `hideConsoleLater`